gemutils/src/common/soap/GEMSOAPToolBox.cc

```diff
@@ -306,11 +315,16 @@ bool gem::utils::soap::GEMSOAPToolBox::sendCommandWithParameter(std::string cons
     xoap::SOAPBodyElement paramValue = env.getBody().addBodyElement(soapcmd);
     paramValue.addTextNode(toolbox::toString("%d", parameter));
 
-    // std::string tool;
+    std::string tool;
     // xoap::dumpTree(msg->getSOAPPart().getEnvelope().getDOMNode(),tool);
+    msg->writeTo(tool);
+    INFO("GEMSOAPToolBox::sendCommandWithParameter SOAP msg " << tool);
+    tool.clear();
```

Comment:
This is a repeatedly used construction and could be streamlined as a function

Reply:
Indeed, #151 created to address this